src/lib/Lex.mll

```diff
@@ -46,8 +46,8 @@ let keywords =
     ("hfill", HFILL);
     ("com", COM);
     ("V", V);
+    ("🥦", V);
```

Comment:
We should add a "C type"! 

https://www.youtube.com/watch?v=lZI6FtLhFtI

Reply:
How about :cookie: for the circle? @cangiuli